benchmark_runner/common/oc/oc.py

```diff
@@ -450,14 +450,18 @@ def get_pod(self, label: str, database: str = '', namespace: str = environment_v
 
     @typechecked
     @logger_time_stamp
-    def save_pod_log(self, pod_name: str, database: str = ''):
+    def save_pod_log(self, pod_name: str, database: str = '', type: str = ''):
```

Comment:
Use the same name (`log_type` or `type`) everywhere.  I suggest `log_type` because `type` is too generic.

Reply:
Fixed